src/sass/table/table.scss

```diff
@@ -0,0 +1,66 @@
+@import "../variables/variables";
+
+:root {
+    /* these exclude the 16px padding for top/bottom */
+    --density-compact-cell-height: 36px;
+    --density-default-cell-height: 48px;
+    --density-relaxed-cell-height: 56px;
+}
+
+.table {
+    overflow-x: auto;
+    position: relative;
+}
+
+.table table {
+    border-collapse: collapse;
+    border-spacing: 0;
+    width: 100%;
+}
+
+.table th,
+.table td {
+    // explicit background is needed for sticky columns
+    background-color: var(--color-background-primary);
+    border-bottom: 1px solid var(--color-stroke-subtle);
+    box-sizing: border-box;
+    height: var(--density-default-cell-height);
+    max-width: 400px;
+    min-width: 124px;
+
+    // minimal padding to prevent text from touching the border
+    padding-block: var(--spacing-100);
+    padding-inline: var(--spacing-200);
+}
+
+.table__cell {
+    text-align: left;
+}
+
+.table__cell--numeric {
+    text-align: right;
```

Comment:
Have you tried just setting text align on the `th` and seeing if the cells under it inherit it? I'm seeing some success with this approach...

Reply:
@ianmcburnie , I vaguely recall some browser inconsistencies with that approach. I can try again; will have to do check the browsers we support.

Reply:
Hmm...I don't see that working anywhere. That used to work with the deprecated html attribute, but not with css.
<img width="1134" alt="image" src="https://github.com/user-attachments/assets/9fb7d96c-b150-4cc0-95d0-ec87ec5b433f">

Reply:
It's working for me here in Safari: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table

Reply:
This was a user-error by me. Oops.